hazelcast/src/test/java/com/hazelcast/instance/HazelcastInstanceManagerTest.java

```diff
@@ -43,62 +39,80 @@
 import static org.mockito.Mockito.doThrow;
 
 @RunWith(HazelcastParallelClassRunner.class)
-@Category({QuickTest.class, ParallelTest.class})
+@Category(QuickTest.class)
 public class HazelcastInstanceManagerTest extends HazelcastTestSupport {
 
-    @Test
-    public void testTestHazelcastInstanceFactory() {
-        TestHazelcastInstanceFactory instanceFactory = new TestHazelcastInstanceFactory();
-        final HazelcastInstance instance1 = instanceFactory.newHazelcastInstance();
-        final HazelcastInstance instance2 = instanceFactory.newHazelcastInstance();
-        final HazelcastInstance instance3 = instanceFactory.newHazelcastInstance();
-
+    private HazelcastInstance hazelcastInstance;
```

Comment:
I pulled out the `HazelcastInstance` to shut it down even if an exception is thrown in the tests.